test/extensions/resource_monitors/injected_resource/config_test.cc

```diff
@@ -0,0 +1,38 @@
+#include "envoy/config/resource_monitor/injected_resource/v2alpha/injected_resource.pb.validate.h"
+#include "envoy/registry/registry.h"
+
+#include "common/event/dispatcher_impl.h"
+
+#include "server/resource_monitor_config_impl.h"
+
+#include "extensions/resource_monitors/injected_resource/config.h"
+
+#include "test/test_common/environment.h"
+#include "test/test_common/test_time.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace ResourceMonitors {
+namespace InjectedResourceMonitor {
+
+TEST(InjectedResourceMonitorFactoryTest, CreateMonitor) {
+  auto factory =
+      Registry::FactoryRegistry<Server::Configuration::ResourceMonitorFactory>::getFactory(
+          "envoy.resource_monitors.injected_resource");
+  EXPECT_NE(factory, nullptr);
+
+  envoy::config::resource_monitor::injected_resource::v2alpha::InjectedResourceConfig config;
+  config.set_filename(TestEnvironment::temporaryPath("injected_resource"));
+  DangerousDeprecatedTestTime test_time;
```

Comment:
I also like to live dangerously. But is it possible to use fake time? I don't know if that requires more of @jmarantz work merged first.

Reply:
I don't think fake time is ready yet, but anyway in this test we don't need any specific kind of time. It's just about creating the resource monitor.